discover/infer.go

```diff
@@ -0,0 +1,77 @@
+package discover
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	effx_api "github.com/effxhq/effx-api-v2-go/client"
+	"github.com/effxhq/effx-cli/data"
+	"github.com/thoas/go-funk"
+)
+
+var serviceDirectoryNames = []string{
+	"services",
+	"apps",
+}
+
+var relavantFiles = []string{
+	"go.mod",           // golang/gin
+	"package.json",     // nodejs
+	"requirements.txt", // python/django/flask
+	"pom.xml",          // java/maven
+	"gemfile",          // rails, sinatra
+	"composer.json",    // larvel
+	"build.gradle",     // java spring
+	"mix.exs",          // elixer/phoenix
+}
+
+// DetectServicesFromRelavantFiles detects services based on
+// containing a service-like file (package.json etc)
+func DetectServicesFromRelavantFiles(workdir string, effxFiles []data.EffxYaml, sourceName string) ([]effx_api.DetectedServicesPayload, error) {
```

Comment:
`relevant` is misspelled

Reply:
I'd argue for just removing it. `DetectServicesFromFiles` is more succinct.

Reply:
`succinct` is a good word hahaha